build/yaml/samples/nsx_v1alpha1_vpcnetworkconfigurations.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: nsx.vmware.com/v1alpha1
+kind: VPCNetworkConfiguration
+metadata:
+  name: vpc-network-config1
+  namespace: ns-1
+spec:
+  appliedToNamespaces:
+    - ns-1
+  tier0Gateway: /infra/tier-0s/t0
+  edgeCluster: 2d9df59f-6dc6-4911-8865-21fadc23d4da
+  defaultSubnetPrefixLength: 28
+  nsxtProject: proj-1
+  publicIPV4Pools:
```

Comment:
`publicIPV4Pools` -> IP block, and I remember in last meeting we decided not to identify whether it's v4 or v6, right?

Reply:
I think they mean the VAPI we do not distinguish ipv4 or ipv6.
But for NSX VPC, public_ipv4_blocks/ipv6_blocks are fields needed to specify ipv4 block or ipv6 block.

Reply:
No, I think even in VAPI it is just that the IPBlock type is common for v4 and v6, but we should still have two fields for v4 and v6 (of the IPBlock type) respectively in the network spec?

Reply:
Right. In network spec of VAPI, we still need 2 fields for v4 and v6.
There is another VAPI we added to list all public ipv4 ip blocks, for this VAPI, in the VAPI review meeting, they suggested us not to distinguish ip v4 or v6.  I think we still need to clarify that there are 2 fields v4 and v6 defined for VPC.